error: use of `write!(stdout(), ...).unwrap()`
  --> $DIR/explicit_write.rs:24:9
   |
24 |         write!(std::io::stdout(), "test").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `print!("test")`
   |
   = note: `-D clippy::explicit-write` implied by `-D warnings`

error: use of `write!(stderr(), ...).unwrap()`
  --> $DIR/explicit_write.rs:25:9
   |
25 |         write!(std::io::stderr(), "test").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `eprint!("test")`

error: use of `writeln!(stdout(), ...).unwrap()`
  --> $DIR/explicit_write.rs:26:9
   |
26 |         writeln!(std::io::stdout(), "test").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `println!("test")`

error: use of `writeln!(stderr(), ...).unwrap()`
  --> $DIR/explicit_write.rs:27:9
   |
27 |         writeln!(std::io::stderr(), "test").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `eprintln!("test")`

error: use of `stdout().write_fmt(...).unwrap()`
  --> $DIR/explicit_write.rs:28:9
   |
28 |         std::io::stdout().write_fmt(format_args!("test")).unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `print!("test")`

error: use of `stderr().write_fmt(...).unwrap()`
  --> $DIR/explicit_write.rs:29:9
   |
29 |         std::io::stderr().write_fmt(format_args!("test")).unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `eprint!("test")`

error: use of `writeln!(stdout(), ...).unwrap()`
  --> $DIR/explicit_write.rs:32:9
   |
32 |         writeln!(std::io::stdout(), "test/ntest").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `println!("test/ntest")`

error: use of `writeln!(stderr(), ...).unwrap()`
  --> $DIR/explicit_write.rs:33:9
   |
33 |         writeln!(std::io::stderr(), "test/ntest").unwrap();
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `eprintln!("test/ntest")`

error: aborting due to 8 previous errors

